test/test-resourcetiming-decompression.js

```diff
@@ -96,6 +96,26 @@
                 expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(3)).to.be("script");
                 expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(4)).to.be("css");
                 expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(5)).to.be("xmlhttprequest");
+                // expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(6)).to.be("html");
+                expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(6)).to.be("navigation");
+                expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(7)).to.be("image");
+                expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(8)).to.be("beacon");
+                expect(ResourceTimingDecompression.getInitiatorTypeFromIndex(9)).to.be("fetch");
+                // expect(ResourceTimingDecompression.getInitiatorTypeFromIndex("a")).to.be("iframe");
```

Comment:
Since iframe, subdocument, and frame all map to "a", I'm not sure what's the preferred decompression

Reply:
No strong preference, `frame` or `iframe` probably